omod/src/main/webapp/resources/js/custom/Custom.js

```diff
@@ -1,34 +1,36 @@
-
 var _FINGERPRINT_DATA = "";
 
 function showMessage() {
-        return "This is from javascript";
-    };
+    return "This is from javascript";
+};
 
 function identifyPatient(fingerprintData){
 
-        _FINGERPRINT_DATA = fingerprintData;
-        var xmlhttp;
-        var data = 'no data found';
-        if (window.XMLHttpRequest) {
-             xmlhttp=new XMLHttpRequest();
-        }
-        else {
-            xmlhttp=new ActiveXObject("Microsoft.XMLHTTP");
-        }
-        xmlhttp.onreadystatechange = function() {
-              if (xmlhttp.readyState==4 && xmlhttp.status==200) {
-                     data = xmlhttp.responseText;
-                     console.log(data);
-                     var data1 = JSON.parse("["+data+"]");
-                     updatePatientListTable(data1, 1);
-              }
+    _FINGERPRINT_DATA = fingerprintData;
+    var xmlhttp;
+    var data = 'no data found';
+    if (window.XMLHttpRequest) {
```

Comment:
@aeadara, I think it's better to use it since we have the library available inside jQuery. Unless you know any good reason we should use XHR? :)

My preference would be to go as far as possible with jQuery for compatibility reason.